<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnsembleConfigDialog</class>
 <widget class="QDialog" name="EnsembleConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ensemble Information</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <widget class="QTextEdit" name="ensStructureTextEdit">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export as CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
